packages/protocol/migrations/03_goldwhitelist.ts

```diff
@@ -0,0 +1,6 @@
+module.exports = (deployer: any) => {
```

Comment:
Should we be registering this contract? If so, `deploymentForCoreContract` is your friend

Reply:
It will also allow you to start to populate this list via the callback

Reply:
And consider adding an entry to the migrationsConfig so that the whitelist can be initialized here

Reply:
deploymentForCoreContract expects a Proxy, which might make using self-destroy difficult